dcp_prototype/backend/scripts/convert_metadata.py

```diff
@@ -0,0 +1,100 @@
+"""
+Simple script to convert a DCP/1 matrix service loom to the DCP/2 cell
+metadata.
+
+Modifies the loom file in place and ends up dropping quite a few of the
+fields from DCP/1.
+"""
+
```

Comment:
Looks good.  
I have some suggestions, which you can take or leave depending on what the use case for this script ends up being.

1.   make the bulk of this script a function that can be called by other scripts.    e.g  write a function alled something like convert_loom_metadata. If we had that, then we could reuse the code more easily.

2. What's the benefit to overwriting the files?   Maybe may an option to supply a command line arg for you the output file.

3.  Perhaps use argparse or something similar to parse the command line args.  right now there is no help message or error checking.

4. add ds.close() after done modifying.

Reply:
Thanks, I fixed everything except for (2). I'm kind of reluctant to make a copy because the script is modifying a tiny part of what could be a rather large file.